install-script/install.sh

```diff
@@ -1766,6 +1817,12 @@ if [[ -z "${DOWNLOAD_ONLY}" ]]; then
             exit 1
         fi
 
+        USER_OPAMP_API_URL="$(get_user_opamp_endpoint "${COMMON_CONFIG_PATH}")"
+        if [[ -n "${USER_OPAMP_API_URL}" && -n "${OPAMP_API_URL}" && "${USER_OPAMP_API_URL}" != "${OPAMP_API_URL}" ]]; then
+            echo "You are trying to install with different opamp endpoint than in your configuration file!"
```

Comment:
not sure I understand this error, I thought the endpoint is only populated in the config file if the user defines the argument `OPAMP_API_URL`? Or is this covering a scenario where a user updates the config with the endpoint and also passes in the opamp url argument?

Reply:
If you look at the context around these lines, it's part of a larger function that verifies that the flags the user is passing match what's in the configuration file, if the things in the configuration file are indeed set.

This only applies when the user has created a file named `"${CONFIG_DIRECTORY}/conf.d/common.yaml"`, as per https://github.com/SumoLogic/sumologic-otel-collector-packaging/pull/76/files/3c5cdf0fd1b6dda72700d5a6a708aae727bfd07f#diff-d396e12fe6ca2006601d801b6bb14cda09dda8bf22fc69b921c7e6704074f896R1809

It's unclear to me precisely why this exception is carved out, but it applies to other similar flags as well.

Reply:
ah makes sense, seems consistent with the other flags.